File: Answer/389.找不同.cpp
/*
 * @lc app=leetcode.cn id=389 lang=cpp
 *
 * [389] 找不同
 */

// @lc code=start
class Solution {
public:
    char findTheDifference(string s, string t) {
        if(s.length() == 0) return t[0];
        int i = 0, j = 0, k = 0;
        while(i < t.length())
        {
            j = 0;
            k = 0;
            while(j < s.length())
            {
                if(t[i] == s[j])
                {
                    s[j] = '\0';
                    k = 1;
                    break;
                }
                j++;
            }
            if(k == 0) return t[i];
            i++;
        }
        return t[t.length() - 1];
    }
};
// @lc code=end

File: Answer/389.找不同.cs
/*
 * @lc app=leetcode.cn id=389 lang=csharp
 *
 * [389] 找不同
 */

// @lc code=start
public class Solution {
    public char FindTheDifference(string s, string t) {
        int a = 0, b = 0;
        for(int i = 0; i < s.Length; i++)
        {
            a += (int)s[i];
            b += (int)t[i];
        }
        b += (int)t[s.Length];
        return (char)(b - a);
    }
}
// @lc code=end

